src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -206,7 +214,7 @@ public void onLoaded() {
                 all.add(execs);
                 Futures.addCallback(execs,new FutureCallback<List<StepExecution>>() {
                     @Override
-                    public void onSuccess(@NonNull List<StepExecution> result) {
+                    public void onSuccess(List<StepExecution> result) {
```

Comment:
Revert?

Reply:
I can, they were only added in my earlier PRs though.